backend/src/main/java/com/summerHack/diningTogether/service/UserService.java

```diff
@@ -33,7 +37,7 @@ public class UserService {
     private ModelMapper modelMapper;
     private UserRepository userRepository;
     private ApplicationRepository applicationRepository;
-
+    private EmailVerificationUtilsImpl emailVerificationUtils;
```

Comment:
Usually it should be an interface instead of specific class so the UserService is decoupled from the actual implementation of the util class.